crates/testlang/outputs/cargo/tests/src/query/engine_tests.rs

```diff
@@ -119,7 +119,7 @@ fn common_test_tree() -> Edge {
 fn test_spread() {
     run_query_test(
         &common_test_tree(),
-        "[TreeNode ... @x1 [DelimitedIdentifier] ... @x2 [DelimitedIdentifier] ...]",
+        "[TreeNode @x1 [DelimitedIdentifier] @x2 [DelimitedIdentifier]]",
```

Comment:
I suggest adding a test to make sure that the anchor in  `[A] . [B]` can skip trivia between them .. Maybe we can use `TerminalKind::is_trivia()` API to detect it?

Reply:
I can do this, but this would be another query change. IIUC, the intent for the `.` operator was to match *no nodes*, trivia or otherwise. Are we sure we want the operator skip over trivia?

Reply:
yes, for sure. and possibly fixed tokens as well. adjacency that doesn't at least skip trivia is pretty much useless.

Reply:
Ok, makes sense. I'm worried that it will not be possible to express the *absence of trivia* in a query, but it's probably not something we need. I'll work on the changes.

Reply:
:heavy_check_mark: